packages/management/tkg/bundle/config/values.yaml

```diff
@@ -16,6 +16,33 @@ frameworkPackage:
   tkrServicePackageValues:
     namespace: tkg-system
     createNamespace: false
+  addonsManagerPackageValues:
+    tanzuAddonsManager:
+      namespace: tkg-system
+      createNamespace: false
+      deployment:
```

Comment:
do we need this by default for addons-manager package? we should only make this config configurable and let the package installer decide if they want to configure the addons-manager deployment resource..

Reply:
This should be the default configuration for addons manager package. 
Could you explain a more on how can users provide configurations for tkg composite package? I mean it should be something similar to today's CLI providers, where do we put that code?